tests/general/LineCountBearTest.py

```diff
@@ -40,3 +42,57 @@ def test_exclude_blank_lines(self):
                                 severity=RESULT_SEVERITY.NORMAL,
                                 file='default')],
             filename='default')
+
+    def test_of_range_allowed_for_lines_per_file(self):
+        self.section.append(Setting('max_lines_per_file', 10))
+        self.section.append(Setting('exclude_blank_lines', True))
+        self.section.append(Setting('min_lines_per_file', 5))
+        self.section.append(Setting('ignore_files_regex', ''))
+        self.check_results(
+            self.uut, ['line 1', '', 'line 2', '', 'line 3', 'line 4'],
+            [Result.from_values('LineCountBear',
+                                'This file had 4 lines, which is 1 lines '
+                                'fewer than the minimum required.',
+                                severity=RESULT_SEVERITY.NORMAL,
+                                file='default')],
+            filename='default')
+
+    def test_of_one_element_range(self):
+        self.section.append(Setting('max_lines_per_file', 5))
+        self.section.append(Setting('min_lines_per_file', 5))
+        self.section.append(Setting('exclude_blank_lines', True))
+        self.section.append(Setting('ignore_files_regex', ''))
+        self.check_results(
+            self.uut, ['line 1', 'line 2', '\tline 3', ' line 4 ', ' line 5',
+                       '\n', ' ', '\t', '\t\n', '\t \n'],
+            [],
+            filename='default')
+
+    def test_exceptions_of_default_lines_of_file(self):
+        self.section.append(Setting('exclude_blank_lines', True))
+        self.section.append(Setting('max_lines_per_file', 2))
+        self.section.append(Setting('min_lines_per_file', 1))
+        self.section.append(Setting('ignore_files_regex', '^.*__init__.py$'))
+        self.check_results(
+            self.uut, [''],
+            [Result.from_values('LineCountBear',
+                                'This file had 0 lines, which is 1 lines '
+                                'fewer than the minimum required.',
+                                severity=RESULT_SEVERITY.NORMAL,
+                                file='default')],
+            filename='default')
+        self.check_results(self.uut, [''], [], filename='__init__.py')
+
+    def test_max_limit_is_smaller_than_min_limit(self):
```

Comment:
Write a test case in which you test any of the ignored files ;)

Reply:
I added such test case :) 

`self.check_results(self.uut, [''], [], filename='__init__.py')`

This test case should check that __init__.py file is empty and despite that min_lines_per_file=1 and max_lines_per_file=2 this test case shouldn't write any message.

Reply:
but there is coverage error in the CI.